### PR TITLE
Fix docstring for fallback modules

### PR DESCRIPTION
**Pull request by kmaehashi**
I was fixing a test code to support pytest-xdist but it seems the original TestDocs test is not working as expected.

context: https://github.com/cupy/cupy/pull/5659/files#r691093294

### CODE REVIEW

**Comment by kmaehashi** [comment]
pfnCI, test this please.

**Comment by kmaehashi** [comment]
pfnCI, test this please.

File: tests/cupyx_tests/fallback_mode_tests/test_fallback.py [hunk 1]
```diff
@@ -235,17 +235,18 @@ def test_fallback_methods_array_external_out(self, xp):
...
+    {'object': ['linalg', 'eig']},
 )
 @testing.gpu
 class TestDocs(unittest.TestCase):
 
     @numpy_fallback_equal()
     def test_docs(self, xp):
-        return getattr(self.object, '__doc__')
+        obj = functools.reduce(getattr, self.object, xp)
+        return getattr(obj, '__doc__')
```
Comment (toslunar) [inline tests/cupyx_tests/fallback_mode_tests/test_fallback.py:249]: How about using `operator.attrgetter`?
```
>>> operator.attrgetter('linalg.eig')
operator.attrgetter('linalg.eig')
>>> operator.attrgetter('linalg.eig')(numpy).__doc__[:100]
'\n    Compute the eigenvalues and right eigenvectors of a square array.\n\n    Parameters\n    ---------'
```

Comment (kmaehashi) [reply tests/cupyx_tests/fallback_mode_tests/test_fallback.py:249]: That's neat! I learned a new thing today 😄

**Comment by kmaehashi** [comment]
pfnCI, test this please.

**Review by toslunar — APPROVED** [review]
LGTM

**Comment by asi1024** [comment]
LGTM.

**Pull request merged**